packages/@aws-cdk/aws-events-targets/lib/firehose-stream.ts

```diff
@@ -0,0 +1,47 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export interface FirehoseStreamProps {
+  /**
+   * The message to send to the stream.
+   *
+   * Must be a valid JSON text passed to the target stream.
+   *
+   * @default - the entire CloudWatch event
+   */
+  readonly message?: events.RuleTargetInput;
+}
+
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export class FirehoseStream implements events.IRuleTarget {
```

Comment:
all of the other targets are named by a convention of  `<service><resource` 
i.e. `SnsTopic`, `CodeBuildProject`, `SfnStateMachine`

can we rename this to align with that convention